src/app/services/student-round.service.ts

```diff
@@ -145,14 +145,61 @@ export class StudentRoundService {
     shareReplay(1),
   );
 
+  interactions$: Observable<Interaction[]> =
+  combineLatest([this.sessionService.currentRoundPath$, this.authService.currentUser$]).pipe(
+    switchMap(([path, user]) =>
+      path && user
+        ? this.firebaseService.getStudentInteractions(path, user.uid)
+        : of([]))
+  );
+
+  totalPollen$: Observable<number> = this.interactions$.pipe(
+    map(interactions =>
+      interactions.filter(interaction => !interaction.isNest)
+    ),
+    map(interactions =>
+      interactions.length)
```

Comment:
The two maps here are redundant.
```suggestion
    map(interactions => interactions.filter(interaction => !interaction.isNest).length)
```